### PR TITLE
Redirect after user creation

### PR DESCRIPTION
**Pull request by Thiblef**
[OO-MISC] Redirect to the user edit form after user creation

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/UserFormView.coffee [hunk 1]
```diff
@@ -0,0 +1,26 @@
+###*
+ * @namespace OpenOrchestra
+###
+window.OpenOrchestra or= {}
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/UserFormView.coffee]: can you add this in namspace in window.OpenOrchestra.User
and move it in the folder user ?

File: BackofficeBundle/Resources/public/coffee/UserFormView.coffee [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+###
+class OpenOrchestra.UserFormView extends FullPageFormView
+
+  ###*
+   * On view ready
+  ###
+  onViewReady: ->
+    success = $('.alert-success', @$el).length > 0
+    if @options.submitted && success
+      entityId = $('#oo_user', @$el).first().data('user-id')
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/UserFormView.coffee]: may be use the same mecanic than other entities : add an hidden input which contains the userId, you have an example you the templateType

**Comment by Thiblef** [comment]
@itkg-alavieille @itkg-ngilain @itkg-nanne PR updated

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**